src/pybamm/models/submodels/interface/sei/base_sei.py

```diff
@@ -268,37 +221,24 @@ def _get_standard_concentration_variables(self, variables):
 
         return variables
 
-    def _get_standard_reaction_variables(self, j_inner, j_outer):
+    def _get_standard_reaction_variables(self, j_sei):
         """
         A private function to obtain the standard variables which
         can be derived from the SEI interfacial reaction current
 
         Parameters
         ----------
-        j_inner : :class:`pybamm.Symbol`
-            The inner SEI interfacial reaction current.
-        j_outer : :class:`pybamm.Symbol`
-            The outer SEI interfacial reaction current.
+        j_sei : :class:`pybamm.Symbol`
+            The SEI interfacial reaction current.
 
         Returns
         -------
         variables : dict
             The variables which can be derived from the SEI currents.
         """
         domain, Domain = self.domain_Domain
-        j_inner_av = pybamm.x_average(j_inner)
-        j_outer_av = pybamm.x_average(j_outer)
-        j_sei = j_inner + j_outer
 
```

Comment:
Why is there no jsei_av variable?

Reply:
It is defined later on, in lines 248-253